Tools/cases_generator/analyzer.py

```diff
@@ -522,6 +522,9 @@ def compute_properties(op: parser.InstDef) -> Properties:
         variable_used(op, "PyCell_New")
         or variable_used(op, "PyCell_GET")
         or variable_used(op, "PyCell_SET")
```

Comment:
Not a huge deal, but I don't think these are needed (and they shouldn't ever be added back).

```suggestion
```